.gitlab-ci.yml

```diff
@@ -0,0 +1,42 @@
+stages:
+  - build
+  - trigger
+
+default:
+  interruptible: true
+
+workflow:
+  rules:
+    - if: $CI_EXTERNAL_PULL_REQUEST_IID
+
+trigger_internal_ci:
+  stage: build
+  image: node
+  tags: 
+    - docker-prod
+  variables:
+    DOWNSTREAM_PROJECT_ID: $DOWNSTREAM_PROJECT_ID
+    DOWNSTREAM_PROJECT_NAME: $DOWNSTREAM_PROJECT_NAME
+  script:
```

Comment:
In the **script** section, the REST API is used to create a branch in the SDK gitlab-repo with the same branch name of github PR. 

CI_EXTERNAL_PULL_REQUEST_SOURCE_BRANCH_NAME: special gitlab var available in gitlab-CI. The value of this var is the which is the branch name of the PR.
DOWNSTREAM_PROJECT_ID - project id of SDK stored as an ENV VAR in the gitlab repo so it is not exposed.